mlir/test/Target/LLVMIR/Import/frame-pointer.ll

```diff
@@ -0,0 +1,8 @@
+; RUN: mlir-translate -import-llvm -split-input-file %s | FileCheck %s
+
+define void @frame_pointer_func() "frame-pointer"="non-leaf" {
+  ; CHECK: llvm.func @frame_pointer_func()
+  ; CHECK-SAME: attributes {frame_pointer = 1 : i64}   
```

Comment:
It seems that the EnumAttr is not capable of forcing the assembly format of it to be a string. I suspect that something like the `LinkageAttr`, see `LLVMAttrDefs.td` is required to get nice printing.
Even though I worked with this at some point, I cannot explain to you why this is necessary.

Alternatively, it might be possible to add an assembly format to the existing attr, but I'm not sure this works.